File: BackEnd/mimi/src/main/java/com/mimi/Dto/Party.java
package com.mimi.Dto;

import java.util.Date;
import java.util.List;

import org.springframework.data.annotation.Id;
import org.springframework.data.mongodb.core.mapping.Document;

import lombok.Data;

@Data
@Document(collection = "party")
public class Party {

	@Id
	private String id;

	private List<User> userList;
	private String ptName;
	private String promiseLocation;
	
	// 약속 정보
	private Store promiseStore;
	private Date promiseTime;

}

File: App/app/src/main/java/com/chd/mimitogether/ui/party/dto/Party.kt
package com.chd.mimitogether.ui.party.dto

import com.chd.mimitogether.dto.UserRequest
import java.io.Serializable
import java.util.*

data class Party (
    var id: String,
    var userList: MutableList<UserRequest>,
    var ptName: String,
    var recoList: List<Int>,
    var promiseLocation: String,
    var promiseStore: Store,
    var promiseTime: Date
): Serializable